packages/types-algosdk/index.d.ts

```diff
@@ -539,13 +546,38 @@ export interface Address {
 
 export type TxnBytes = Uint8Array;
 
+export interface SSCParams {
+  'approval-program': string;
+  'clear-state-program': string;
+  creator: string;
+  'global-state': SSCStateSchema[];
+  'global-state-schema': SSCSchemaConfig;
+  'local-state-schema': SSCSchemaConfig;
+}
+
 export interface AccountAssetInfo {
   amount: number;
   'asset-id': number;
   creator: string;
   'is-frozen': boolean;
 }
 
+export interface CreatedApps {
+  id: number;
+  params: SSCParams;
+}
+
+export interface CreatedAssets {
+  index: number;
+  params: SSCParams;
+}
+
+export interface AppLocalState {
+  id: number;
+  'key-value': SSCStateSchema[];
+  schema: SSCSchemaConfig;
+}
+
 export interface AccountInfo {
```

Comment:
There is no such structure in `algosdk`.
How about renaming it to `AccountState`? It would make more sense in my opinion.

Reply:
This was present by before. But i think this structure is present in `algosdk`.
```
const accountInfo = await deployer.algodClient.accountInformation(account).do();
```
Changing name to `AccountState`.

Reply:
good, thanks.